tests/reg_tests/test_DVConstraints.py

```diff
@@ -1,24 +1,31 @@
+# Standard Python modules
 import os
 import unittest
-import numpy as np
+
+# External modules
 from baseclasses import BaseRegTest
-from pygeo import DVGeometry, DVConstraints
-from stl import mesh
+import numpy as np
 from parameterized import parameterized_class
+from stl import mesh
+
+# First party modules
+from pygeo import DVConstraints, DVGeometry
 
 try:
+    # External modules
     import geograd  # noqa: F401
 
-    missing_geograd = False
+    geogradInstalled = True
 except ImportError:
-    missing_geograd = True
+    geogradInstalled = False
 
 try:
+    # First party modules
     from pygeo import DVGeometryMulti
```

Comment:
Importing the class no longer raises `ImportError`, so this check is not doing anything and the test skipping is not working. We need to check for pySurf instead like you did for others optional modules like GeoGrad above.

Reply:
That's a good catch, I missed it during my review :grin:

Reply:
Good catch, I also missed this.